src/advanced-concepts/custom_signature_validation.md

```diff
@@ -0,0 +1,17 @@
+# Custom Signature Validation Scheme
+
+Account Abstraction on Starknet supports various signature schemes. This means that signature schemes on Starknet are not limited to just one, any standard signature scheme can be validated, for example Starknet signature, Secp256k1, Secp256r1, Eip191 et al are some of the custom signatures that can be validated on Starknet currently. 
+
+### The Concepts of Accounts and Signers
+
+i. **Account:** All accounts are smart contracts that can hold assets and execute transactions on Starknet protocol, these account contracts however must implement some specific methods outlined in SNIP-6.
+
+ii. **Signers:** These are responsible for digitally signing transactions and provide the authorization needed to initiate transactions. 
+Digital signatures are cryptographic proofs that transactions are authorized by corresponding accounts.
+
+In summary, Starknet accounts are normal blockchain accounts that hold assets and initiate transactions onchain, while signers provide the authorization required to ensure that transactions originating from these accounts are secure, valid and executed.
```

Comment:
Maybe put more emphasis on the fact that signers are **offchain**, and that the implementation of the validation functions of SNIP-6 dictate how to "link" signers onchain, by verify the signatures in account contracts. You can compare it with EOA in Ethereum, and explains that it adds more flexibility.

With great power comes great responsibility, so a small advisory on potential security issues with improper validation implementation would be great as well.

Reply:
I would still think that an emphasis on the offchain part of signers is missing